src/Traction/Control.hs

```diff
@@ -73,7 +74,7 @@ class MonadIO m => MonadDb m where
 instance MonadDb Db where
   liftDb = id
 
-instance MonadDb m => MonadDb (ExceptT e m) where
+instance MonadDb m => MonadDb (EitherT e m) where
```

Comment:
Not sure this is right, downstream is exceptT (at least an be)

Reply:
(I mean from a doc/error message perspective)

Reply:
Probably should be the more general `ExceptT` type for maximum compatibility and better documentation.

If I change this back to `ExceptT` and it still works (as I expect) with `transformers-either` then I think that is a better solution.